.drone.yml

```diff
@@ -107,6 +107,28 @@ pipeline:
       event: [push, tag]
       branch: [dev]
 
+  compile-gource-video:
+    repo: kowalatech/kcoin
+    secrets: [ docker_username, docker_password ]
+    dockerfile: gource/Dockerfile
+    commands:
+      - make gource-video
```

Comment:
This make task uses docker, which isn't necessary when the pipeline uses a gource docker image. I guess there needs to be a new make task that just runs `gource`, and assumed that everything is installed to make that happen (which should be the case because it will run inside an appropriate docker image in the CI).